eng/Versions.props

```diff
@@ -122,6 +122,10 @@
     <!-- Dependencies from https://github.com/NuGet/NuGet.Client -->
     <NuGetBuildTasksPackageVersion>6.8.1-rc.2</NuGetBuildTasksPackageVersion>
   </PropertyGroup>
+  <PropertyGroup>
+    <!-- Dependencies from https://github.com/dotnet/msbuild -->
+    <MicrosoftBuildPackageVersion>17.8.5</MicrosoftBuildPackageVersion>
```

Comment:
Does automation update this? If not, how are we keeping it up-to-date?

Reply:
Yup, this move us to using the codeflow version which was already in the xml file but not used in the props:
https://github.com/dotnet/installer/blob/marcpopMSFT-updatemsbuildversion/eng/Version.Details.xml#L158